members-service/triggerChampaignMemberUpdate.js

```diff
@@ -10,18 +10,26 @@ const logger = new OperationsLogger({
   client: new DocumentClient(),
 });
 
-export async function handler(e, ctx, cb, fn = updateMember) {
-  const [item] = e.Records;
+async function update(item, fn = updateMember) {
   const record = Converter.unmarshall(item.dynamodb.NewImage);
-  if (!updateMemberEvent(record)) return cb(null, 'Not a member update event');
+
+  if (!updateMemberEvent(item.eventName, record)) {
+    console.error('ERROR: Not a member update event');
```

Comment:
Is this an actually an error?
If I understand correctly this function will be triggered whenever a new record is written in the 'operations table'. So, if a record that's not of type "updateMemberEvent" is added, we'd be logging an Error, when actually nothing has gone wrong.